src/main/scala/Validator.scala

```diff
@@ -74,7 +75,25 @@ class Validator(var tableCsvFile: URI, sourceUri: String = "") {
             ""
           )
         }
-        val warningsAndErrors = table.validateRow(row)
+        val (
+          warningsAndErrors,
+          primaryKeyValues,
+          foreignKeyReferenceValues,
+          foreignKeyValues
+        ) = table.validateRow(row)
+        val initialAllPrimaryKeyValuesSize = allPrimaryKeyValues.size
+        if (allPrimaryKeyValues.contains(primaryKeyValues)) {}
```

Comment:
todo: implement the contains check instead of the sizes check.

Reply:
Done